### PR TITLE
merry holidays to those who celebrate

### PR DESCRIPTION
**Pull request by pillowtrucker**
I recently had to fix the same thing with another wgpu/winit thing so I figured might as well do this too

### CODE REVIEW

File: examples/render-opengl/src/opengl.rs [hunk 1]
```diff
@@ -40,12 +41,18 @@ pub fn launch_opengl_window() -> Result<App, Box<dyn Error>> {
...
-		.with_preference(ApiPrefence::FallbackEgl)
+		.with_preference(ApiPreference::FallbackEgl)
 		.with_window_builder(Some(window_builder))
 		.build(&events, <_>::default(), |configs| {
 			configs
-				.filter(|c| c.srgb_capable())
-				.max_by_key(|c| c.num_samples())
+				.filter(|c| match c {
+					Config::Egl(c) => c.srgb_capable(),
+					_ => false,
```
Comment (Speykious) [inline examples/render-opengl/src/opengl.rs]: This should be `true` in the default case, otherwise it'll filter everything out and get the following error:
```
thread 'main' panicked at examples/render-opengl/src/opengl.rs:56:18:
called `Option::unwrap()` on a `None` value
```

**Review by Speykious — CHANGES_REQUESTED** [review]
The WGPU example doesn't work right now and it seems like dependencies weren't all updated. I'm gonna try to see what I can do.

In any case thank you so much :D

**Comment by pillowtrucker** [comment]
hmm wgpu definitely both builds and works fine for me, not sure what could be wrong there. wasm32 target might not work fully as it's painful to test, so my only test for that is whether it compiles

**Comment by Speykious** [comment]
The `rwh_05` feature was missing

**Comment by Speykious** [comment]
> wasm32 target might not work fully as it's painful to test, so my only test for that is whether it compiles

`trunk serve` doesn't work for you?

File: examples/render-opengl/src/opengl.rs [hunk 2]
```diff
@@ -40,12 +41,18 @@ pub fn launch_opengl_window() -> Result<App, Box<dyn Error>> {
...
-				.filter(|c| c.srgb_capable())
-				.max_by_key(|c| c.num_samples())
+				.filter(|c| match c {
+					Config::Egl(c) => c.srgb_capable(),
+					Config::Glx(c) => c.srgb_capable(),
+				})
+				.max_by_key(|c| match c {
+					Config::Egl(c) => c.num_samples(),
+					Config::Glx(c) => c.num_samples(),
+				})
```
Comment (Speykious) [inline examples/render-opengl/src/opengl.rs]: So why this change? I just tried and `c.srgb_capable()` is still available on the root data structure. Same for `c.num_samples()`. Now it breaks on Mac and Windows.

Comment (pillowtrucker) [reply examples/render-opengl/src/opengl.rs]: not in my version. I feel like we're testing on different libraries or compilers. I'm fixing up the other platforms rn

Comment (pillowtrucker) [reply examples/render-opengl/src/opengl.rs]: I think I had the wrong version of the import

Comment (Speykious) [reply examples/render-opengl/src/opengl.rs]: Ah, turns out it was a wrong import, alright :v

**Review by Speykious — APPROVED** [review]

**Comment by Speykious** [comment]
Merry Christmas :D
